docker-compose.yml

```diff
@@ -9,3 +9,21 @@ services:
       - .:/app
     environment:
       FLASK_ENV: development
+      DATABASE_PARAMS: Driver={ODBC Driver 17 for SQL Server};Server=tcp:db,1433;Database=sqldb-web-prod-001;Uid=sa;Pwd=Cl1mat3m1nd!;Encrypt=no;TrustServerCertificate=no;Connection Timeout=30;
+      FLASK_APP: "climatemind.py"
+    command: ["./wait-for-it", "mysql:1433", "--", "python3", "climatemind.py"]
```

Comment:
The “mysql:1433” part should not say MySQL, it should say something like "vc-db:1433" or “db:1433” I think. Not big deal but can be misleading otherwise. maybe “mssql-db:1433” would work, just need to try it before changing.